bin/parse_phenotype.py

```diff
@@ -37,6 +37,24 @@
 assessments = ["BPQ", "TAS", "AQ", "EQ", "STAI",
     "GAD7", "MAIA", "PANAS", "PHQ9", "UCLA", "GSQ"]  # known assessments
 
+BL_one = ["participant_id", "BL_Track_meanacc", "BL_Track_RHO", "BL_Track_meanconf", "BL_Time_meanacc", "BL_Time_RHO",
+       "BL_Time_meanconf", "BL_Disc_meanacc", "BL_Disc_ROC", "BL_Disc_meanconf", "BL_d_prime", "BL_ITPE_T", "BL_ITPE_D", BL_C]
```

Comment:
`BL_C` should be a string? Careful with your variable type.

Reply:
What you are describing here is what comments in code are for. It would be more obvious for a reviewer to see the comments in the code, not in the commit message.
These things do stand out but they stand out for the unintended reason (syntax error). Commit messages are more like footnotes. If it's something super important that you want to bring to attention, write it in the message on PR, not the commit message.